charts/alfresco-activiti-admin/values.yaml

```diff
@@ -82,6 +82,12 @@ database:
       # -- Key within the secret holding the database password for admin app
       password: DATABASE_PASSWORD
 
+workaround:
+  # -- Enable workaround for https://hyland.atlassian.net/browse/ACTIVITI-5113
+  # Set to `true` to enable the workaround regardless of the appVersion
+  # Set to `false` to disable the workaround (or any other value)
+  ACTIVITI-5113: true
```

Comment:
I am not sure if this is a good value name. Maybe `activiti-entrypoint` would be more descriptive. It would give some idea on what type of workaround is this.

Reply:
I think it's difficult to describe the workaround itself so linking to the jira seemed like a good enough choice.